.gitignore

```diff
@@ -4,7 +4,6 @@
 .settings/
 
 # PHP prj
-build/
```

Comment:
You are right, there is no need to remove the `build` from`.gitignore`. Will fix